bugsnag-android-performance/src/test/java/com/bugsnag/android/performance/internal/SpanBlockingTest.kt

```diff
@@ -131,9 +130,7 @@ class SpanBlockingTest {
         assertFalse(span.isBlocked())
         assertTrue(spanProcessor.isEmpty())
 
-        assertNull(condition1!!.upgrade())
-
-        condition1.close()
+        condition1!!.close()
```

Comment:
I'm not sure I see how this amended test now covers the amended logic and aim to allow canceled spans to be upgraded?